lstchain/io/io.py

```diff
@@ -767,23 +774,20 @@ def check_metadata(metadata1, metadata2):
         assert metadata1[k] == metadata2[k]
 
 
-def global_metadata(source, input_url=""):
+def global_metadata():
     """
     Get global metadata container
 
     Returns
     -------
     `lstchain.io.lstcontainers.MetaData`
     """
-    if source:
-        input_url = source.input_url
 
     metadata = MetaData()
     metadata.LSTCHAIN_VERSION = lstchain.__version__
     metadata.CTAPIPE_VERSION = ctapipe.__version__
     metadata.CTAPIPE_IO_LST_VERSION = ctapipe_io_lst.__version__
     metadata.CONTACT = "LST Consortium"
-    metadata.SOURCE_FILENAMES.append(os.path.basename(input_url))
```

Comment:
is this variable not used anymore?

Reply:
This was the point of this PR, right? To no store it in the metadata, because it might be to large.

Reply:
but for single files shouldn't this still be used?

Reply:
@rlopezcoto
This variable will not be used anymore.
For single files this var contained just the name of the file, this is useless.